docs/template_usage.rst

```diff
@@ -155,7 +155,7 @@ A demonstration in a screen capture:
 Embed ipywidgets
 ----------------
 
-Other ipywidgets can be embedded by setting them in a trait and adding widget_serialization, accessing them in the with the `jupyter-widget` tag:
+Any ipywidget can be embedded by setting them in a trait and adding widget_serialization, accessing them in the template with the `jupyter-widget` tag:
```

Comment:
```suggestion
Any ipywidget can be embedded by setting them in a trait and adding widget_serialization, accessing them in the template with the ``jupyter-widget`` tag:
```

I think single quote are .md symbols, you need to use 2 of them in rst files.

Reply:
Thanks for the suggestion, it is committed.